lib/rules/id-length.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @fileoverview Rule that warns when identifier names are shorter or longer
+ *               than the values provided in configuration.
+ * @author Burak Yigit Kaya aka BYK
```

Comment:
Not true that copyright automatically implies restriction of use of code. Copyright just means that this particular code snippet is definitely owned (intellectually) by you. It is possible to use a software license which asserts copyright but allows distribution and modification under reasonable conditions. Check out the project's license for more information.